webrtc.html

```diff
@@ -6866,19 +6879,16 @@ <h3 id="sec.identity-verify-assertion">Verifying Identity Assertions</h3>
     pc.ontrack = function (evt) {
         if (evt.track.kind === "audio") {
           if (answerer) {
-            // TODO: Figure out how to reuse an existing m-line with something like:
-            // audio = evt.transceiver;
-            // audio.sender.activate();
-            // This avoids the need for an extra re-offer.
-            // Similarly for video.
-            audio = pc.addTransceiver(evt.track.kind);
+            audio = evt.transceiver;
```

Comment:
as far as I can tell, the `RTCTrackEvent` interface doesn't have a `transceiver` attribute.